cirq/testing/random_superposition.py

```diff
@@ -0,0 +1,20 @@
+import numpy as np
+
+def normalize(array: np.ndarray):
+    norm = np.linalg.norm(array)
+    return array / norm
+
+def random_superposition(size: int) -> np.ndarray:
+    """
+    Args:
+        size: Specified size returns a 2^size length array.
+    Returns:
+        Normalized random array.
+    """
+    state_vector = np.array([])
+    for _ in range(0, 2**size):
+        state_vector = np.append(
+            np.array([complex(np.random.standard_normal(1),
+                              np.random.standard_normal(1))]),
+            state_vector)
+    return normalize(state_vector)
```

Comment:
Instead of looping, just tell numpy how many entries you want:
```
state_vector = np.random.normal(size=n)
state_vector += 1j * np.random.normal(size=n)
state_vector /= np.linalg.norm(state_vector)
return state_vector
```

Reply:
Done.